frontend/src/components/Dashboard/Main/Carousel.jsx

```diff
@@ -76,13 +76,13 @@ class Carousel extends Component {
                         }}/>
                 }
 
-                Your Top Track{this.props.topTracks !== null && this.props.topTracks !== undefined && this.props.topTracks.length !== 0 ? this.props.topTracks.items.length > 1 ? 's' : '' : ''}:
+                Your Top Track{this.props.topTracks !== null && this.props.topTracks !== undefined && this.props.topTracks.length !== 0 ? this.props.topTracks.length > 1 ? 's' : '' : ''}:
```

Comment:
an easier way to check for this would be 

```javascript
`Your Top Track ${this.props.topTracks && this.props.topTracks.length > 1 && 's'}
```
because the && operator checks if a variable evaluates to a truthy value and null/undefined are both falsy values